build/build.go

```diff
@@ -95,12 +95,14 @@ type CallFunc struct {
 }
 
 type Inputs struct {
-	ContextPath      string
-	DockerfilePath   string
-	InStream         *SyncMultiReader
-	ContextState     *llb.State
-	DockerfileInline string
-	NamedContexts    map[string]NamedContext
+	ContextPath          string
+	DockerfilePath       string
+	InStream             *SyncMultiReader
+	ContextState         *llb.State
+	DockerfileInline     string
+	NamedContexts        map[string]NamedContext
+	DockerfileMappingSrc string
```

Comment:
Add a comment that these are filled in by the builder

Reply:
Added :)